### PR TITLE
Add spanish translation

### PR DESCRIPTION
**Pull request by BrianCraig**
Hi! I made this PR for adding Spanish translation that Jester Requested on Discord, and seems to be the fix for #42 

I tested on my PC and everything seems working fine, Here is the end result: https://imgur.com/a/1RRXpWQ

### CODE REVIEW

File: src/languages/es_ES.json [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+      "6": "<0>Práctica</0> es para pruebas, donde ambos capitánes son controlados por el mismo usuario."
+    },
+    "basics": {
+      "title": "Resúmen",
+      "1": "$t(instructions.home.1)",
+      "2": "$t(instructions.home.2)"
+    },
+    "hostAndGuestTurns": {
+      "title": "Turnos de anfitrión y invitado",
+      "1": "Los turnos tienen una acción específica (<1>ELEGIR</1>, <3>BLOQUEAR</3>, <5>VETAR</5>) y algunos posibles modificadores.",
```
Comment (lalitpatel) [inline src/languages/es_ES.json:150]: @HSZemi Should we translate `PICK`, `BAN` and `SNIPE` in this sentence or leave them as it is.

Comment (HSZemi) [reply src/languages/es_ES.json:150]: We should either translate them nowhere or everywhere. Currently, the english words PICK, BAN, and SNIPE are being used everywhere.

Comment (lalitpatel) [reply src/languages/es_ES.json:150]: My suggestion is to keep it the same throughout the instructions and across languages.

Comment (BrianCraig) [reply src/languages/es_ES.json:150]: Ok, also i think if we mantain this words in the Help section, also on the Draft it should have the same verbs.
`Esperando a el otro capitán a elegir…`  should be `Esperando a el otro capitán a pickear…`  .

Its important to clarify that we use the word "pickear" as the action of pick,"banear", and "snipear" as the actions to ban and snipe, so i suggest using them when needed, or we can use simply "pick/ban/snipe" in that sentences to remain more consistent on using the same words. 

What do you think about this ?

Comment (lalitpatel) [reply src/languages/es_ES.json:150]: > Ok, also i think if we mantain this words in the Help section, also on the Draft it should have the same verbs.
> `Esperando a el otro capitán a elegir…` should be `Esperando a el otro capitán a pickear…` .

Agree.

> Its important to clarify that we use the word "pickear" as the action of pick,"banear", and "snipear" as the actions to ban and snipe, so i suggest using them when needed, or we can use simply "pick/ban/snipe" in that sentences to remain more consistent on using the same words.

I support your suggestion to clarify the meaning of the words pick/ban/snipe in Spanish. I feel that it is fine to use translation of the verbs in the sentences but not in the actual colored tags. The colored tags should always say `PICK`, `BAN` and `SNIPE` for the sake of consistency.

**Comment by HSZemi** [comment]
Looks good to me! Thank you for your contribution :yellow_heart:

**Pull request merged**